inc/customizer/controls/react/src/icon-color/IconColor.js

```diff
@@ -0,0 +1,16 @@
+import ColorControl from '../common/ColorControl';
```

Comment:
There shouldn't be new files inside the `src/` folder unless they are inside the `repeater` directory. Each folder inside `/src/` represents a fully-fledged customizer control, but the IconColor isn't one 😄